tests/core/test_external_search.py

```diff
@@ -158,11 +158,12 @@ def test_setup_index_creates_new_index(
         # The alias hasn't been passed over to the new index.
         alias = "test_index-" + external_search_fixture.search.CURRENT_ALIAS_SUFFIX
         assert alias == external_search_fixture.search.works_alias
+        print(current_index, alias)
         assert True == external_search_fixture.search.indices.exists_alias(
-            current_index, alias
+            alias, index=current_index
```

Comment:
Did the param order here change between the ES and OS library? Were there any other changes we should be watching out for?

Reply:
Yes the signature changed between the libraries, this is the only code change that occured.